substreams-helper/src/storage.rs

```diff
@@ -34,12 +124,317 @@ impl Uint256 {
     }
 }
 
+impl Default for Uint256 {
```

Comment:
Does BigInt have a default impled on it already? If so you can just derive Default for the struct rather than manually impling it

Reply:
Doesn't seem like it:
```
the trait bound `substreams::scalar::BigInt: Default` is not satisfied
the trait `Default` is not implemented for `substreams::scalar::BigInt
```

Will keep it in mind for the next times though.